crates/topos/src/components/subnet/commands/subnet.rs

```diff
@@ -0,0 +1,17 @@
+use clap::{ArgGroup, Args, Parser};
+use std::path::PathBuf;
+
+#[derive(Parser, Debug)]
+#[clap(about = "Run Polygon Edge", trailing_var_arg = true)]
+pub struct Run {
+    /// Installation directory path for Polygon Edge binary.
+    /// If not provided, Polygon Edge binary will be expected in the current directory
+    #[arg(long, env = "TOPOS_SUBNET_PATH", default_value = ".")]
```

Comment:
maybe precising here in the env variable name that's about a path to a binary

```suggestion
    #[arg(long, env = "TOPOS_SUBNET_BIN_PATH", default_value = ".")]
```

```suggestion
    #[arg(long, env = "TOPOS_POLYGON_EDGE_BIN_PATH", default_value = ".")]
```